### PR TITLE
clicking email takes you to Profile

### PR DESCRIPTION
**Pull request by cheemurakami**
Resolves https://github.com/gdomingu/crystal_swap/issues/29

Screen Shot 2020-08-10 at 3 32 56 PM

Clicking email takes you to profile page.

### CODE REVIEW

File: app/javascript/landing/Layout.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { UserContext } from "../context/UserContext";
 import { Link as RouterLink } from "react-router-dom";
 import useMediaQuery from "@material-ui/core/useMediaQuery";
 import { useTheme } from "@material-ui/core/styles";
+import { nominalTypeHack } from "prop-types";
```
Comment (gdomingu) [inline app/javascript/landing/Layout.js:18]: What is this?

File: app/javascript/landing/Layout.js [hunk 2]
```diff
@@ -63,6 +68,9 @@ const homebutton = () => {
   );
 };
 
+
```
Comment (gdomingu) [inline app/javascript/landing/Layout.js:71]: remove extra spaces please.

File: app/javascript/landing/Layout.js (hunk 1 above)
Comment (cheemurakami) [reply app/javascript/landing/Layout.js:18]: I don't know.

Comment (gdomingu) [reply app/javascript/landing/Layout.js:18]: Please remove if it was an accident!

**Pull request merged**